### ISSUE TITLE
[TRIGGER] Server-Sent Events

### ISSUE THREAD

**Issue by fiatjaf** [PipedreamHQ/roadmap#441]
Is it feasible to trigger Pipedream workflows from [SSE](https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events/Using_server-sent_events) events? Maybe not as you would have to keep a connection open to each service (not sure if that is a big burden or not), but here's the idea.

**Comment by dylburger** [comment]
@fiatjaf not directly. Is there a specific service you’re trying to integrate via SSE right now or were you just curious?

**Comment by fiatjaf** [comment]
Yes, etleneum.com, my own service. Too small so I wasn't going to mention it.

Besides that I had been thinking if SSE wouldn't fit better many of the services out there that currently send out webhooks. I thought that was just a fantasy of mine but then I saw (after opening this issue) that Pipedream does exactly this with workflow results and even arbitrary workflow events. Maybe this is a trend now.

**Comment by dylburger** [comment]
I just checked out your API docs - very cool service.

Yeah, we personally like the simplicity of SSE, and since we're optimizing for uni-directional event pipelines it made sense (vs. a WebSocket API, for example).
